django/utils/http.py

```diff
@@ -176,10 +176,16 @@ def parse_http_date(date):
     try:
         year = int(m.group('year'))
         if year < 100:
-            if year < 70:
-                year += 2000
-            else:
-                year += 1900
+            # two-digit RFC850-date format year to be
+            # parsed relative to current year in accordance with RFC7231
```

Comment:
This RFC is already mentioned in the docstring, please remove this comment.

Reply:
Agree, one comment should be enough. Would it make sense to rewordthe top comment to something like this - making the fact it's about being able to parse two digit years explicit?
```
    # email.utils.parsedate() does the job for RFC1123 dates; however
    # we implement our own RFC-compliant parsing as RFC7231 makes it mandatory
    # to also support two-digit RFC850 dates.
```

Reply:
RFC 850 is not only about two-digit years but also about long name for weekdays, etc. so I don't think that we should change this comment.

Reply:
OK!